src/classes/Pantry.js

```diff
@@ -1,6 +1,10 @@
 class Pantry {
   constructor(pantry) {
-    this.ingredientsInPantry = pantry || [];
+    let copyPantry = undefined;
```

Comment:
Don't think we need all this since we removed the empty pantry test - I also fixed this in my PR so we will likely run into a merge conflict if we merge both of them